Runtime/Scripts/Sensor.cs

```diff
@@ -14,45 +14,50 @@ public class Sensor : MonoBehaviour
         /// <summary>
         /// Serial of the sensor to handle events from.
         /// </summary>
+        [Tooltip("Serial number of the physical sensor.")]
         public string Serial = "";
         /// <summary>
+        /// Radius of the field of view of the sensor. SensMax TAC-B 
+        /// sensors have a 10m radius.
+        /// </summary>
+        [Tooltip("Radius of the visual indicator showing the sensor's field of view. Does not effect the function of this script. SensMax TAC-B sensors have a 10m radius.")]
```

Comment:
```suggestion
        [Tooltip("Radius of the visual indicator showing the sensor's field of view. Has no effect on the function of this script. SensMax TAC-B sensors have a 10m radius.")]
```
Syns setningen bare var litt rar.